.github/workflows/_mac-test.yml

```diff
@@ -91,6 +91,12 @@ jobs:
           name: ${{ inputs.build-environment }}
           use-gha: true
 
+      - name: Download TD artifacts
+        continue-on-error: true
+        uses: ./.github/actions/download-td-artifacts
+        with:
+          use-gha: true
```

Comment:
I don't think you need this though, as I remember we have setup the permission to S3 on macOS runners

Reply:
The earlier step for downloading build artifacts in mac also uses gha

Reply:
The step is probably redundant now, but it's ok to fix them separately